xrpl4j-client/src/main/java/org/xrpl/xrpl4j/client/XrplClient.java

```diff
@@ -493,6 +497,20 @@ public AccountInfoResult accountInfo(AccountInfoRequestParams params) throws Jso
     return jsonRpcClient.send(request, AccountInfoResult.class);
   }
 
+  /**
+   * Return AccountNftsResult for an {@link Address}.
+   *
+   * @param account to get the NFTs for.
+   *
+   * @return {@link AccountNftsResult} containing list of accounts for an address.
+   * @throws JsonRpcClientErrorException If {@code jsonRpcClient} throws an error.
+   */
+  public AccountNftsResult accountNfts(Address account) throws JsonRpcClientErrorException {
+    return this.accountNfts(
```

Comment:
Please add preconditions to assert non-null inputs on any functions.

```suggestion
    Objects.requireNonNull(account);
    return this.accountNfts(
```